modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -309,82 +390,296 @@ export default class Ecdsa {
    * @returns {SignConvertRT}
    */
   signConvert(shares: SignConvert): SignConvertRT {
-    let shareParticipant: BShare | GShare, shareToBeSend: AShare | MUShare;
+    let shareParticipant: Partial<BShare> | Partial<GShare>, shareToBeSent: Partial<AShare> | MUShare;
     let isGammaShare = false;
+    let kShare: Partial<KShare> = {};
     if (shares.xShare && shares.yShare && shares.kShare) {
```

Comment:
this case is never called, can we just remove this? or should we start calling `signConvert` more properly?